### DISCUSSION TITLE
Do the debris-related commands work?

### DISCUSSION THREAD

**Discussion by ghost** [mastercoms/tf2cfg#611] [Q&A] [answered]
https://github.com/mastercomfig/mastercomfig/blob/develop/config/mastercomfig/cfg/comfig/comfig.cfg
```
//fx_drawimpactdebris 0 // Disable bullet surface debris effect
//fx_drawimpactdust 0 // Disable bullet surface dust effect
//fx_drawmetalspark 0 // Disable bullet surface spark effect
```
The variables aren't detected by the game console.

**Comment by mastercoms** [comment, marked as answer]
Yes, they are unlocked by a dxsupport modification which is included in the mastercomfig package
